README.md

```diff
@@ -124,3 +124,22 @@ synchronization:
   roles:
     - linux-system-roles.timesync
 ```
+
+Install and configure chrony with multiple NTP servers and custom advanced 
+settings: log ```measurements```,```statistics``` and ```tracking```
+into ```/var/log/chrony```:
```

Comment:
Please use the single backtick for inline text that should be preformatted.  You only need the 3 backtick form for multiline code/preformatted text.